docs/references/remotes.md

```diff
@@ -0,0 +1,283 @@
+>
+>    - secure_serialize: bool (deprecated)
+>
+>    - requester_pays : bool, or str default False
+>        Whether to use requester-pays requests. This will include your
+>        project ID `project` in requests as the `userProject`, and you'll be
+>        billed for accessing data from requester-pays buckets. Optionally,
+>        pass a project-id here as a string to use that as the `userProject`.
+>
+>    - session_kwargs: dict
+>        passed on to ``aiohttp.ClientSession``; can contain, for example,
+>        proxy settings.
+>
+>    - endpoint_url: str
+>        If given, use this URL (format protocol://host:port , *without* any
+>        path part) for communication. If not given, defaults to the value
+>        of environment variable "STORAGE_EMULATOR_HOST"; if that is not set
+>        either, will use the standard Google endpoint.
+>
+>    - default_location: str
+>        Default location where buckets are created, like 'US' or 'EUROPE-WEST3'.
+>        You can find a list of all available locations here:
+>        https://cloud.google.com/storage/docs/locations#available-locations
+>
+>    - version_aware: bool
+>        Whether to support object versioning. If enabled this will require the
+>        user to have the necessary permissions for dealing with versioned objects.
+
+### Azure Blob Storage
+
+DataChain uses [adlfs](https://fsspec.github.io/adlfs/) to interact with Azure Blob Storage. Authentication can be achieved by using any of the method described at [adlfs documentation](https://github.com/fsspec/adlfs?tab=readme-ov-file#setting-credentials). You can also pass the following configuration parameters to the adlfs filesystem as client_config dictionary.
+
+
+>	- account_name: str
+>        The storage account name. This is used to authenticate requests
+>        signed with an account key and to construct the storage endpoint. It
+>        is required unless a connection string is given, or if a custom
+>        domain is used with anonymous authentication.
+>
+>    - account_key: str
+>        The storage account key. This is used for shared key authentication.
+>        If any of account key, sas token or client_id is specified, anonymous access
+>        will be used.
+>
+>    - sas_token: str
+>        A shared access signature token to use to authenticate requests
+>        instead of the account key. If account key and sas token are both
+>        specified, account key will be used to sign. If any of account key, sas token
+>        or client_id are specified, anonymous access will be used.
+>
+>    - request_session: Session
+>        The session object to use for http requests.
+>
+>    - connection_string: str
+>        If specified, this will override all other parameters besides
+>        request session. See
+>        http://azure.microsoft.com/en-us/documentation/articles/storage-configure-connection-string/
+>        for the connection string format.
+>
```

Comment:
```suggestion
- `connection_string`: `str` (default: `None`)

    If specified, this will override all other parameters besides
    request session. See
    http://azure.microsoft.com/en-us/documentation/articles/storage-configure-connection-string/
    for the connection string format.

```